packages/core/src/oidc/init.ts

```diff
@@ -58,7 +57,7 @@ export default function initOidc(
     signed: true,
   } as const);
 
-  const oidc = new Provider(issuer, {
+  const oidc = new Provider(envSet.oidc.issuer, {
```

Comment:
Maybe add a comment about why should not use a de-struct here.  I'm afraid we may loose the context later.

Reply:
Added a comment at the first occurrence of the `envSet` usage